templates/performance_optimized.j2

```diff
@@ -17,11 +17,19 @@ op_defaults \
 # production HANA
 #
 
+{% if data.platform == "gcp" %}
+primitive rsc_ip_{{ sid }}_HDB{{ instance }} ocf:heartbeat:IPaddr2 \
+    params \
+    ip="{{ data.virtual_ip }}" cidr_netmask="32" nic="eth0" \
+    op start timeout="20" op stop timeout="20" \
+    op monitor interval="10" timeout="20"
+{% else %}
 primitive rsc_ip_{{ sid }}_HDB{{ instance }} ocf:heartbeat:IPaddr2 \
     params \
     ip="{{ data.virtual_ip }}" cidr_netmask="{{ data.virtual_ip_mask }}" \
```

Comment:
@juadk if the only difference need is the cidr_netmask, I would consider an inline IF here to check the platform. This way, it would avoid repeating the entire block.  Another point is, for GCP is not needed to inform the nic into the resource? (AFAIK it assumes eth0 as default, but maybe to be more visually readable, I would include it here too).

Reply:
Agree with @diegoakechi 

I would do something like that:
cidr_netmask="{{ 32 if data.platform == "gcp" else data.virtual_ip_mask }}"

About the nic, if this interface is the same than the used by crm, consider that libvirt might uses eth1